src/bios_handler.cpp

```diff
@@ -286,6 +286,43 @@ void IbmBiosHandler::saveAmmToVpd(const std::string& i_memoryMirrorMode)
     // TODO: Call write keyword API to update the value in VPD.
 }
 
+void IbmBiosHandler::saveAmmToBios(const std::string& i_ammVal)
+{
+    if (i_ammVal.size() != constants::VALUE_1)
+    {
+        std::cerr << "Bad size for AMM received, Skip writing to BIOS"
```

Comment:
Why `std::cerr` is being used here? Other APIs are using `logging::logMessage()`

Reply:
Fixed.